website/docs/Contribute.md

```diff
@@ -120,7 +120,23 @@ Tests are automatically run via GitHub actions. There are two workflows:
 1. [build.yml](https://github.com/microsoft/autogen/blob/main/.github/workflows/build.yml)
 1. [openai.yml](https://github.com/microsoft/autogen/blob/main/.github/workflows/openai.yml)
```

Comment:
We should make another PR to document the contrib and contrib-openai workflows @rickyloynd-microsoft @thinkall @BeibinLi @kevin666aa @IANTHEREAL @gagb @afourney

Reply:
IMO, it makes sense to have a utility function (`getModelForTest(Enum)`) that is a single entry point for all tests requiring OpenAI model interaction. This file will list all models available to tests (as enums). All test that want to make OpenAI calls will got and ask for a model from there.

If a new model is added (e.g. GPT-4 vision) a maintainer would need to got to this model registry file and add the model name there (rather than create own config_list from env, file. etc. which is fragile).

This file can also be a central switchboard determining if a model is available and if not - have OpenAI test skips with the reason 'No XYZ OpenAI model/endpoint is configured'.

This way it is easier to both - (1) run and maintain the tests and (2) make a simple contrib guide